src/styles/blocks/catalog.scss

```
@@ -17,6 +17,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all $transition-time ease;
```

It's better to specify which properties you transition directly. I'll approve, but please add this change
```suggestion
    transition: color, transform $transition-time ease;
```